File: basic_samples/v9/sample.py
import requests

key = "Insert your Speechace API key here"
api_endpoint = "Insert your Speechace API endpoint here" 
url = api_endpoint + "/api/scoring/text/v9/json"
dialect = "en-us"
user_id = "81ozow"

url += '?' + 'key=' + key + '&dialect=' + dialect + '&user_id=' + user_id

payload ={'text':'apple'};
user_file_handle = open('apple.wav', 'rb')

files = {'user_audio_file': user_file_handle}
response = requests.post(url, data=payload, files=files)
print(response.text)

File: basic_samples/v9/sample.php
<?php

$key="Insert your Speechace API key here";
$api_endpoint="Insert your Speechace API endpoint here";
$ch = curl_init($api_endpoint."/api/scoring/text/v9/json?user_id=1234&dialect=en-us&key=".$key);
$opts = array(
    CURLOPT_POST => 1,
    CURLOPT_HTTPHEADER => array("Content-Type:multipart/form-data"),
    CURLOPT_POSTFIELDS => array(
        "text" => "banana",
        "user_audio_file" => curl_file_create("apple.wav"),
    ),
    CURLOPT_RETURNTRANSFER => true
);

curl_setopt_array($ch,$opts);
$raw = curl_exec($ch);

if(curl_errno($ch) > 0) {
    echo("There was an error using the speechace api: ".curl_error($ch));
}
else {
    // var_dump($raw);

    // Print the raw json response
    print_r($raw);

    // Print the PHP decoded json response
    // print_r(json_decode($raw));
}

curl_close($ch);
?>
